<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BugReportDialog</class>
 <widget class="QDialog" name="BugReportDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>448</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>480</width>
    <height>448</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>480</width>
    <height>448</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Bug report</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#leTitleBug
{
padding-left: 10px;
height: 36px;
width: 432px;
}

#teDescribeBug
{
height: 116px;
width: 432px;
}

#lTitle, #lDescribeBug
{
font-weight: 600;
}


</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <property name="spacing">
    <number>5</number>
   </property>
   <property name="topMargin">
    <number>5</number>
   </property>
   <item>
    <widget class="QWidget" name="wHeaderBug" native="true">
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QLabel" name="lDescInfoBug">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>Please clearly describe the issue you encountered. The more details, the easier for us to resolve. Your submission will be reviewed by our development team.</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="wTitleBug" native="true">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="lTitle">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>Title</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="leTitleBug">
        <property name="inputMask">
         <string/>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="maxLength">
         <number>150</number>
        </property>
        <property name="placeholderText">
         <string>Optional</string>
        </property>
        <property name="type" stdset="0">
         <string notr="true">mega</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="wDescribeBug" native="true">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="lDescribeBug">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>Describe the problem</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextEdit" name="teDescribeBug">
        <property name="cursor" stdset="0">
         <cursorShape>IBeamCursor</cursorShape>
        </property>
        <property name="type" stdset="0">
         <string notr="true">mega</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="wCheckBox" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <property name="spacing">
          <number>5</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>19</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QCheckBox" name="cbAttachLogs">
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="text">
            <string>Attach diagnostic log files to assist with debugging</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
           <property name="type" stdset="0">
            <string notr="true">mega</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="wButtons" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>219</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="bCancel">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>36</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>36</height>
         </size>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="text">
         <string>Cancel</string>
        </property>
        <property name="type" stdset="0">
         <string notr="true">outline</string>
        </property>
        <property name="dimension" stdset="0">
         <string notr="true">medium</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="bSubmit">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>36</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>36</height>
         </size>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="text">
         <string>Submit</string>
        </property>
        <property name="default">
         <bool>true</bool>
        </property>
        <property name="type" stdset="0">
         <string notr="true">primary</string>
        </property>
        <property name="dimension" stdset="0">
         <string notr="true">medium</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
